linode/manager
# upcoming: [M3-7861]: Include powered-off data in clone analytics event

## Description 📝
Updates the Create Linode analytics event when cloning to include whether the source Linode was powered-off.

## Target release date 🗓️
4/15

## How to test 🧪

- Enable the Linode Clone UI Feature flag
- Go to http://localhost:3000 and turn on analytics logs for Adobe: 
   - Ensure `REACT_APP_ADOBE_ANALYTICS_URL` is set in your .env
   - In the browser console, type `_satellite.setDebug(true)`
- Clone a powered-on/off Linode
- Verify the Create Linode event is fired and the `data.isLinodePoweredOff` field reflects the power status of the origin Linode
- Optional (it takes about ~1 hr for Adobe dashboards to update): 
   - Log into Adobe Analytics
   - Go to the Projects folder and open the Linode Cloud - Dev Dashboard. Scroll down to the bottom two graphs, "Component Details vs. Component Clicks" and "Number of Events With Linodes Powered Off" and observe the data before and ~1 hour after your testing, which should now include metrics for your event(s) that fired.

## As an Author I have considered 🤔

*Check all that apply*


---
